tester_app/app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,53 @@
+// Core variables and mixins
+@import 'bootstrap/variables';
```

Comment:
We need a to include that file defining the image-url here!

and this should go in the tutorial like this as well.

Maybe actually, we should separate this into 2 files. The sass helper, and the bootstrap variables file, and the bootstrap variables file loads the sass helper! It has to be that way, b/c we need to import the sass helper before loading bootstrap, I think.